benchmark/main.cpp

```diff
@@ -56,979 +50,1198 @@
-                     i * BUFSIZE / 1024 );                                     \
-} while( 0 )
-
-#if defined(MBEDTLS_MEMORY_BUFFER_ALLOC_C) && defined(MBEDTLS_MEMORY_DEBUG)
-
-#define MEMORY_MEASURE_INIT                                        \
-    size_t max_used, max_blocks, max_bytes;                        \
-    size_t prv_used, prv_blocks;                                   \
-    mbedtls_memory_buffer_alloc_cur_get( &prv_used, &prv_blocks ); \
-    mbedtls_memory_buffer_alloc_max_reset( );
-
-#define MEMORY_MEASURE_PRINT( title_len )                          \
-    mbedtls_memory_buffer_alloc_max_get( &max_used, &max_blocks ); \
-    for( i = 12 - title_len; i != 0; i-- ) mbedtls_printf( " " );  \
-    max_used -= prv_used;                                          \
-    max_blocks -= prv_blocks;                                      \
-    max_bytes = max_used + MEM_BLOCK_OVERHEAD * max_blocks;        \
-    mbedtls_printf( "%6u heap bytes", (unsigned) max_bytes );
 
-#else
-#define MEMORY_MEASURE_INIT
-#define MEMORY_MEASURE_PRINT( title_len )
-#endif
+static void alarm() {
+    alarmed = 1;
+}
 
-#define TIME_PUBLIC( TITLE, TYPE, CODE )            \
-do {                                                \
-    unsigned long ms;                               \
-    int ret = 0;                                    \
-    Timer t;                                        \
-    MEMORY_MEASURE_INIT;                            \
-                                                    \
-    mbedtls_printf( HEADER_FORMAT, TITLE );         \
-    fflush( stdout );                               \
-                                                    \
-    t.start();                                      \
-    CODE;                                           \
-    t.stop();                                       \
-    ms = t.read_ms();                               \
-                                                    \
-    if( ret != 0 )                                  \
-    {                                               \
-        PRINT_ERROR;                                \
-    }                                               \
-    else                                            \
-    {                                               \
-        mbedtls_printf( "%6lu ms/" TYPE, ms );      \
-        MEMORY_MEASURE_PRINT( sizeof( TYPE ) + 1 ); \
-        mbedtls_printf( "\r\n" );                   \
-    }                                               \
-} while( 0 )
-
-static int myrand( void *rng_state, unsigned char *output, size_t len )
-{
+static int myrand(void *rng_state, unsigned char *output, size_t len) {
     size_t use_len;
     int rnd;
 
-    if( rng_state != NULL )
+    if (rng_state != NULL) {
         rng_state  = NULL;
+    }
 
-    while( len > 0 )
-    {
+    while (len > 0) {
         use_len = len;
-        if( use_len > sizeof(int) )
+        if(use_len > sizeof(int)) {
```

Comment:
Add a space after `if`.

Reply:
I think this review comment is based on an old commit. Please note that later on there is a commit that fixes these style problems by running astyle with the Mbed OS configuration file.